recipes/msdf-atlas-gen/all/CMakeLists.txt

```
@@ -0,0 +1,10 @@
+cmake_minimum_required(VERSION 3.10)
+project(cmake_wrapper)
+
+include(conanbuildinfo.cmake)
+conan_basic_setup(TARGETS)
```

Let's revert this then..
```suggestion
conan_basic_setup()
```

---

Yep ^^